src/reanimated2/core.js

```diff
@@ -226,10 +226,15 @@ export const runOnJS = (fun) => {
 const capturableConsole = console;
 runOnUI(() => {
   'worklet';
+  // global._setGlobalConsole({
+  //   log: runOnJS(capturableConsole.log),
+  //   warn: runOnJS(capturableConsole.warn),
+  //   error: runOnJS(capturableConsole.error),
+  // });
   const console = {
     log: runOnJS(capturableConsole.log),
     warn: runOnJS(capturableConsole.warn),
     error: runOnJS(capturableConsole.error),
   };
-  _globalSetter('console', console); // eslint-disable-line
+  _setGlobalConsole(console); // eslint-disable-line
```

Comment:
Why you used `eslint-disable-line` here?
if it complains about not resolving `_setGlobalConsole` then you should use `/* global _setGlobalConsole */`, like in updateProps file.